### PR TITLE
The Capitoline Triad and support

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

**Comment by Hanmac** [comment]
Probably need AI logic too?

**Comment by tool4ever** [comment]
that's why it's marked just like _Fabrication Foundry_ 🤷‍♂️

File: forge-gui/src/main/java/forge/player/HumanCostDecision.java [hunk 1]
```diff
@@ -255,12 +255,19 @@ public PaymentDecision visit(final CostExile cost) {
...
         }
         boolean totalCMC = false;
+        boolean totalCMCgreater = false;
         String totalM = "";
         if (type.contains("+withTotalCMCEQ")) {
             totalCMC = true;
             totalM = type.split("withTotalCMCEQ")[1];
             type = TextUtil.fastReplace(type, TextUtil.concatNoSpace("+withTotalCMCEQ", totalM), "");
         }
+        if (type.contains("+withTotalCMCGE")) {
```
Comment (Hanmac) [inline forge-gui/src/main/java/forge/player/HumanCostDecision.java:265]: isn't the same part needed there in CostExile?

https://github.com/Card-Forge/forge/blob/8fe5bb2086029b60a1cf7d4de40dbf33d7975f36/forge-game/src/main/java/forge/game/cost/CostExile.java#L193-L197

**Comment by Agetian** [comment]
I can look into the AI for both cards once this PR is in :)

**Review by Hanmac — APPROVED** [review]

**Pull request merged**